converter/SEMSinput.py

```diff
@@ -138,7 +138,7 @@ def process_row(row):
 
         sql = "insert into %s values (%s)" % (table_def['name'], ",".join(value_placeholders))
 
-        values = row[1:]
+        id, *values = row
```

Comment:
You need to actually use `id` in place of `row[0]` too. Sorry, I wasn't explicit about that.

Reply:
I think it works either way, but you're right that this will be cleaner, so will do that now.

Reply:
It does work either way, but I wrote the initial comment to improve the clarity of the parts of `row`.

Reply:
yep, totally agreed :)